app/components/Header/Header.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
+import { Text, TouchableOpacity, View } from 'react-native'
+import { Actions } from 'react-native-router-flux'
+
+import styles from './styles'
+
+import ImageButton from '../ImageButton'
+
+class ExtraActionsMenu extends Component {
+	render() {
+		if (this.props.display) {
+			return (
+				<View style={styles.extraActionsContainer}>
+					{this.props.actions.map(function({label, onPress}, key){
+						return (
+							<View key={key}>
+								<TouchableOpacity onPress={onPress}>
+									<Text style={styles.action}>
+										{label}
+									</Text>
+								</TouchableOpacity>
+							</View>
+						)
+					})}
+				</View>
+			)
+		} else {
+			return (
+				<View></View>
+			)
+		}
+	}
+}
+
+ExtraActionsMenu.propTypes = {
+	actions: React.PropTypes.array,
+	display: React.PropTypes.bool,
+}
+
+class Header extends Component {
+	constructor() {
+		super()
+		this.state = {
+			displayExtraActionsMenu: false,
+		}
+	}
+
+	render() {
+		return (
+			<View style={styles.mainContainer}>
+				<View style={styles.headerContainer}>
+					<View style={styles.navContainer}>
+						{(() => {
+							if (this.props.navigation) {
+								return (
+									this.props.navigation.map(function({source, onPress}, key){
```

Comment:
Je me suis dit qu'on aurait pu avoir besoin de plusieurs boutons éventuellement. But no useless code admitted, you're right